### ISSUE TITLE
Default colour choice

### ISSUE THREAD

**Issue by tomchadwin** [NathanW2/ogrcompare#11]
I'm not sure red and green are quite right. The behaviour of this tool is not the same as a diff, so I think using a common diff/Github colour scheme is not desirable. I wonder whether no colour (ie black on white, or whatever the design is) would be better for matches, and red for differences.

What do others think?

**Comment by NathanW2** [comment]
Yeah main thing is just showing what is different.  Doesn't really matter how. So maybe only highlight non matching.
